### PR TITLE
chore: initial my collection store tests #trivial

### PR DESCRIPTION
**Pull request by brainbicycle**
The type of this PR is: **Chore**

This PR resolves [CX-704]

### Description

- Adds some basic functionality tests for MyCollectionNavigationModel and MyCollectionArtworkModel

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-704]: https://artsyproduct.atlassian.net/browse/CX-704

### CODE REVIEW

File: src/lib/Scenes/MyCollection/State/__tests__/MyCollectionArtworkModel-tests.tsx [hunk 1]
```diff
@@ -1,5 +1,232 @@
...
+  getGeminiCredentialsForEnvironment,
+  uploadFileToS3,
+} from "lib/Scenes/Consignments/Submission/geminiUploadToS3"
+
+const getConvectionGeminiKeyMock = getConvectionGeminiKey as jest.Mock<any>
+const getGeminiCredentialsForEnvironmentMock = getGeminiCredentialsForEnvironment as jest.Mock<any>
+const uploadFileToS3Mock = uploadFileToS3 as jest.Mock<any>
+
 describe("MyCollectionArtworkModel", () => {
-  it("works", () => {
```
Comment (damassi) [inline src/lib/Scenes/MyCollection/State/__tests__/MyCollectionArtworkModel-tests.tsx:2]: Wow @brainbicycle, did you do anything to get these tests to work? For whatever reason, when I tried to write tests I couldn't trigger store updates -- any change would come back as the initial state.

**Review by damassi — APPROVED** [review]
Looks great and 👍👍 on how easy it is to test easy peasy!

File: src/lib/Scenes/MyCollection/State/__tests__/MyCollectionArtworkModel-tests.tsx (hunk 1 above)
Comment (brainbicycle) [reply src/lib/Scenes/MyCollection/State/__tests__/MyCollectionArtworkModel-tests.tsx:2]: It seemed to just work for me, I am not sure what changed here but it seems to be working now 🤷

Comment (damassi) [reply src/lib/Scenes/MyCollection/State/__tests__/MyCollectionArtworkModel-tests.tsx:2]: 😅

**Review by ds300 — APPROVED** [review]
looks great, thanks!

**Pull request merged**